src/style.css

```diff
@@ -1 +1,57 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.logo__img {
+  margin: 10px 0 10px 50px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
```

Comment:
Please provide the same padding-left instead of using margin-left for image
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/941ae39e-69ae-4002-9a15-7891111af974)